### DISCUSSION TITLE
How to avoid the bubble on the boundary using MultiSmoothCircleIC

### DISCUSSION THREAD

**Discussion by wenpeng231024** [idaholab/moose#17201] [Q&A Modules: Phase field] [answered]
Hi Moose group,

Now I try to use "MultiSmoothCircleIC" to build multiple bubbles sample.  But I found there are some bubbles on the boundary. Because my boundary condition is not a periodic boundary, do you know how to avoid the bubble on the boundary?

Thanks,

Peng

**Comment by laagesen** [comment, marked as answer]
Right now it is only set up assuming periodic BCs. If you want to try modifying the code to avoid having bubbles intersect boundaries, I can suggest how it can be done. Another option would be to use SpecifiedSmoothCircleIC, then you can place the bubbles exactly where you want them.

**Reply by wenpeng231024** [reply]
Got it. I will try to use SpecifiedSmoothCirclesIC to place the bubbles. Thanks very much.

Peng
